# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../emacs-xemacs-shared/no-shared.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xemacs-21.4.12/lib-src/Makefile.in.in	2002-08-20 13:34:16.000000000 +0200
+++ xemacs-21.4.12-fixed/lib-src/Makefile.in.in	2003-03-29 03:20:01.000000000 +0100
@@ -238,19 +238,7 @@
 ## Install the internal utilities.  Until they are installed, we can
 ## just run them directly from lib-src.
 ${archlibdir}: all
-	@echo; echo "Installing utilities run internally by XEmacs."
-	./make-path ${archlibdir}
-	if test "`(cd ${archlibdir} && $(pwd))`" != "`$(pwd)`"; then \
-	  for f in ${PRIVATE_INSTALLABLE_EXES}; do \
-	    (cd .. && $(INSTALL_PROGRAM) lib-src/$$f ${archlibdir}/$$f) ; \
-	  done ; \
-	fi
-	if test "`(cd ${archlibdir} && $(pwd))`" \
-	     != "`(cd ${srcdir}     && $(pwd))`"; then \
-	  for f in ${PRIVATE_INSTALLABLE_SCRIPTS}; do \
-	    (cd .. && $(INSTALL_PROGRAM) ${srcdir}/$$f ${archlibdir}/$$f); \
-	  done ; \
-	fi
+	@echo; echo "Not installing utilities run internally by XEmacs."
 
 install: ${archlibdir}
 	@echo; echo "Installing utilities for users to run."
